packages/common/src/SolidityTestUtils.ts

```diff
@@ -158,3 +160,42 @@ export async function mineTransactionsAtTime(
     await startMining(web3);
   }
 }
+
+// Very similar to the function above, but uses a hardhat network to send hardhat node compatible RPC methods.
+export async function mineTransactionsAtTimeHardhat(
```

Comment:
nice

Reply:
it might be nice if there was just one method and it switched the kind of call sent depending on the network so that we don't have to have this extra function name.

Reply:
not a big issue for now though.

Reply:
Is there a concrete way to tell what node you're using?